contractsV5/KyberNetwork.sol

```diff
@@ -730,18 +722,30 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
         require(success, "Transfer platform fee to taker platform failed");
         emit HandlePlatformFee(tradeData.input.platformWallet, tradeData.platformFeeWei);
 
-        // Send total fee amount to fee handler with reserve data.
-        require(
-            feeHandlerContract.handleFees.value(tradeData.networkFeeWei)(eligibleWallets, rebatePercentages),
-            "Transfer network fee to FeeHandler failed"
-        );
+        //no need to handle fees if no fee paying reserves
+        if (tradeData.numFeePayingReserves == 0) return true;
+
+        // create array of reserves receiving fees + fee percent per reserve
```

Comment:
fix comment.
create array of rebate wallets

Reply:
done